spec/services/organization_update_service_spec.rb

```diff
@@ -68,6 +115,28 @@
         expect(organization.partners.map { |p| p.profile.enable_quantity_based_requests })
           .to eq([false, false])
       end
+
+      it "should NOT update partners' request types when enabling request types on the organization" do
+        organization.update!(enable_quantity_based_requests: false)
+        organization.update!(enable_quantity_based_requests: true)
```

Comment:
This is required to satisfy [a condition in the service](https://github.com/rubyforgood/human-essentials/blob/58814c53241a34c1d06f668c558752f7c266602f/app/services/organization_update_service.rb#L21)

Reply:
This doesn't seem to be the right way to do this. Updating the exact same field twice on a specific model wouldn't (shouldn't) result in any change to behavior. If it is, something is coded wrong. If we're relying on some kind of side effect / callback here, we should instead make it explicit.

Reply:
I'll chenge the business logic, this should fix this issue in the tests